.github/workflows/test.yml

```diff
@@ -148,6 +148,44 @@ jobs:
       - name: Upload coverage
         uses: codecov/codecov-action@v2
 
+  long-3-test:
```

Comment:
Not the best practice, we should use template for this purpose. We will do it in the separate task I guess

Reply:
ok. will remove this then